src/microbe_stage/Microbe.cs

```diff
@@ -518,9 +532,15 @@ public void Damage(float amount, string source)
         }
         else if (source == "chunk")
         {
+            PlaySoundEffect("res://assets/sounds/soundeffects/microbe-toxin-damage.ogg");
+
             // Divide damage by physical resistance
             amount /= Species.MembraneType.PhysicalResistance;
         }
+        else if (source == "atpDamage")
+        {
+            PlaySoundEffect("res://assets/sounds/soundeffects/microbe-release-toxin.ogg");
```

Comment:
Also a TODO comment here for replacing this sound in the future should be added.